tests/Integration/Behaviour/Features/Scenario/Product/update_prices_without_states.feature

```diff
@@ -0,0 +1,280 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags update-prices-without-states
+@restore-all-tables-before-feature
+@reboot-kernel-before-feature
+@clear-cache-before-feature
+@update-product-prices
+@update-prices-without-states
+@reboot-kernel-after-feature
+@clear-cache-after-feature
+Feature: Update product price fields from Back Office (BO).
+  As a BO user I want to be able to update product fields associated with price.
```

Comment:
The description of the feature should explain the purpose of those tests Because honestly, I wonder what is the purpose here since all the scenarios look very much like the ones from `update_prices.feature` except for the tax rule related to France but does it really change so much?

Not that having too many behat tests is really an issue anyway 😅